### DISCUSSION TITLE
https

### DISCUSSION THREAD

**Discussion by gbvitrano** [opendatasicilia/opendatasicilia.it#54] [Generale]
Attualmente il sito funziona sia [https://](https://opendatasicilia.it/) che in [http://](http://opendatasicilia.it/)
da cpanel dovrebbe essere possibile reindirizzare http in https cosi mandiamo in pensione definitivamente http
Che ne dite?

**Comment by gbvitrano** [comment]
Fatto... ho aggiunto le regole al file .htaccess

**Reply by aborruso** [reply]
Gianni, le cose che hanno un impatto di codice o di configurazione apriamole o poi spostiamole come issue per favore.

Aggiungi qui una nota con il path del file, e le righe che hai aggiunto e/o modificato.

Grazie

**Comment by gbvitrano** [comment]
Il file .htaccess si trova nella cartella radice del sito `public_html`
Per reindirizzare http in https  normalmente vanno aggiunte queste righe
```
RewriteEngine on
RewriteCond %{HTTPS} off
RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
```

ma nel nostro .htaccess era gia prensente `RewriteEngine on`
ho aggiunto solo 
```
RewriteCond %{HTTPS} off
RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
```

**Comment by gbvitrano** [comment]
risolto da @gabacode #57
